site/assets/js/snippets.js

```diff
@@ -16,36 +16,44 @@
 (() => {
   'use strict'
 
+  // Boosted mod: Try not using `const` at the first layer in this file, otherwise Storybook will fail render some examples.
+
+  // storybook-start Overview
```

Comment:
It would be more consistent with the other "shortcode tags" to have `// storybook-start overview` (no uppercase for the first letter) isn't it?